components/frontend/src/dashboard/MetricSummaryCard.js

```diff
@@ -72,27 +72,10 @@ export function MetricSummaryCard({ header, onClick, selected, summary, maxY })
     const chartProps = {
         animate: animate,
         colors: colors,
-        events: [
-            {
-                childName: "all",
-                target: "data",
-                eventHandlers: {
-                    onClick: () => {
-                        return [
-                            {
-                                target: "labels",
-                                mutation: () => {
-                                    return { active: false } // Clear tooltip before filtering
```

Comment:
No longer needed?

Reply:
Yes, this is made obsolete by not rendering the tooltip in the portal.